Apsitvarkom.Api/Controllers/PollutedLocationController.cs

```diff
@@ -43,7 +52,7 @@ public async Task<ActionResult<IEnumerable<PollutedLocationGetResponse>>> GetAll
     [ProducesResponseType(StatusCodes.Status400BadRequest, Type = typeof(List<string>))]
     [ProducesResponseType(StatusCodes.Status500InternalServerError)]
     [HttpGet("All/OrderedByDistance")]
-    public async Task<ActionResult<IEnumerable<PollutedLocationGetResponse>>> GetAll([FromQuery] CoordinatesGetRequest coordinates)
+    public async Task<ActionResult<IEnumerable<PollutedLocationGetResponse>>> GetAll([FromQuery] CoordinatesCreateRequest coordinates)
```

Comment:
This is `CoordinatesCreateRequest` because it seemed logical to reuse the same object that is in the `Create` flow. I argue that we're essentially "creating" coordinates here (validation is the same, required fields)